src/gui.c

```diff
@@ -4,6 +4,7 @@
  * Copyright (C) 2009-2015 Keldon Jones
  *
  * Source file modified by B. Nordli, August 2015.
+ * Source file modified by J.-R. Reinhard, October 2016.
```

Comment:
Around lines 3828 and 7119: GUI tooltip is incorrect when combining Contact Specialist with Xeno worlds. See ALIEN check in gui.c, lines 3828 and 7119. (Check "GUI Options -> Display costs during placement", have Contact Specialist active, enough military to settle a Xeno military world, and then hover over it during placement.)